<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="pay1lap-71-PC" name="Three" tests="9" failures="1" timestamp="16 May 2017 21:49:27 GMT" time="135.802" errors="0">
  <testcase name="getSalesmanBalance" time="3.389" classname="pay1_distributor.SalesmanTransfer"/>
  <testcase name="stransfer" time="36.601" classname="pay1_distributor.SalesmanTransfer"/>
  <testcase name="verifySalesmanBalance" time="1.537" classname="pay1_distributor.SalesmanTransfer"/>
  <testcase name="lastTransactions" time="3.627" classname="pay1_distributor.SalesmanTransfer"/>
  <testcase name="mainReport" time="4.552" classname="pay1_distributor.SalesmanTransfer"/>
  <testcase name="salesman_listchanges" time="33.158" classname="pay1_distributor.SalesmanTransfer"/>
  <testcase name="spullback" time="11.06" classname="pay1_distributor.SalesmanTransfer"/>
  <testcase name="accHistory1" time="18.902" classname="pay1_distributor.SalesmanTransfer">
    <skipped/>
  </testcase> <!-- accHistory1 -->
  <testcase name="accHistory1" time="22.39" classname="pay1_distributor.SalesmanTransfer">
    <failure type="java.lang.AssertionError" message="Error while confirming pullback transaction">
      <![CDATA[java.lang.AssertionError: Error while confirming pullback transaction
	at pay1_distributor.SalesmanTransfer.accHistory1(SalesmanTransfer.java:422)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- accHistory1 -->
</testsuite> <!-- Three -->
